UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentAncillaryFiles.cs

```diff
@@ -169,5 +200,30 @@ public bool CreateProductFile(string batchId, string exchangeSetInfoPath, string
                 return false;
             }
         }
+
+        private string CrcString(string fullFilePath)
+        {
+            var crcHash = Crc32CheckSumProvider.Instance.Compute(fileSystemHelper.ReadAllBytes(fullFilePath));
+            return crcHash.ToString();
+        }
+
+        private string GeIsdtUadt(SalesCatalogueDataProductResponse salescatalogProduct)
```

Comment:
Kindly change method name to GetIssueAndUpdateDate

Reply:
Resolved